internal/controller/runtime/test_client_obj_tracker.go

```diff
@@ -16,49 +16,75 @@ import (
 type CustomTracker struct {
 	clienttesting.ObjectTracker
 	shootSequence []*gardener_api.Shoot
-	callCnt       int
+	seedSequence  []*gardener_api.Seed
+	shootCallCnt  int
+	seedCallCnt   int
 	mu            sync.Mutex
 }
 
-func NewCustomTracker(tracker clienttesting.ObjectTracker, shoots []*gardener_api.Shoot) *CustomTracker {
+func NewCustomTracker(tracker clienttesting.ObjectTracker, shoots []*gardener_api.Shoot, seeds []*gardener_api.Seed) *CustomTracker {
 	return &CustomTracker{
 		ObjectTracker: tracker,
 		shootSequence: shoots,
+		seedSequence:  seeds,
 	}
 }
 
-func (t *CustomTracker) GetCallCnt() int {
-	return t.callCnt
-}
-
 func (t *CustomTracker) IsSequenceFullyUsed() bool {
-	return t.callCnt == len(t.shootSequence) && len(t.shootSequence) > 0
+	return (t.shootCallCnt == len(t.shootSequence) && len(t.shootSequence) > 0) && t.seedCallCnt == len(t.seedSequence)
 }
 
 func (t *CustomTracker) Get(gvr schema.GroupVersionResource, ns, name string) (runtime.Object, error) {
```

Comment:
Have you considered writing unit tests for this `Get` and `getNextObject` methods?

Reply:
Thanks for hint, I will add those unit test